### PR TITLE
Setup for new libraries

### PR DESCRIPTION
**Pull request by core23**
After the packagist setup, we should merge this 

Refs: https://github.com/sonata-project/dev-kit/issues/348

### CODE REVIEW

File: config/projects.yml [hunk 1]
```diff
@@ -139,14 +139,9 @@ datagrid-bundle:
         symfony: ['2.8', '3.2', '3.3', '3.4']
 
 doctrine-extensions:
-  excluded_files:
-    - README.md
-  docs_target: false
   branches:
     master:
       php: ['7.1', '7.2']
-    1.x:
```
Comment (Soullivaneuh) [inline config/projects.yml]: Wow, what the hell is going on here? :-D

We can't just remove an already existing package and start an empty one.

Why do we have this: https://github.com/sonata-project/sonata-doctrine-extensions with `sonata-project/doctrine-extensions` as package name and now we have https://github.com/sonata-project/doctrine-extensions ? This makes no sense and will be very confusing.

Comment (core23) [reply config/projects.yml]: You're right. Think we can remove the new https://github.com/sonata-project/doctrine-extensions repo and rename the existing one (?).

Comment (Soullivaneuh) [reply config/projects.yml]: @core23 I can rename repositories, but not delete them.

**Comment by core23** [comment]
I just removed the doctrine-extensions repo, so we can start working on the twig and form repo.

Can you merge this @Soullivaneuh @greg0ire ?

**Comment by greg0ire** [comment]
@core23 I_have_no_power_here.jpg

**Comment by OskarStark** [comment]
> You're right. Think we can remove the new https://github.com/sonata-project/doctrine-extensions repo and rename the existing one (?).

I archived this repo as a first step. If you like I can delete it

**Pull request merged**